### ISSUE TITLE
[TW-1025] Greater than one recurring task broken in 1.9.4?

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1060]
_steve rader on 2011-01-29T09:10:02Z says:_ 

 In all cases I've tried, the N>1 "task ls" invocations are wedged after
creating two recurring tasks...

<pre>

  $ \rm -rf ~/.task
  $ task add recur:1month due:02/05/2011 Pay the mortgage
  Created task 1.

  $ task add recur:1month due:02/05/2011 Pay the second mortgage
  Created task 1.

  $ task ls

  ID Proj Prio Age Description
   1             - Pay the mortgage
   2             - Pay the second mortgage

  2 tasks
  $ task ls
  The duration '-' was not recognized.
  $ task ls
  The duration '-' was not recognized.

</pre>

Further, the "task list" and "task long" invocation indicate the 2nd "parent"
is incorrectly spawning a child each time the report is ran..

<pre>

  $ \rm -rf ~/.task
  $ task add recur:1month due:02/05/2011 Pay the mortgage
  Created task 1.

  $ task add recur:1month due:02/05/2011 Pay the second mortgage
  Created task 1.

  $ task list

  ID Project Pri Due      Active Age Description
   1             2/5/2011          - Pay the mortgage
   2             2/5/2011          - Pay the second mortgage

  2 tasks
  $ task list

  ID Project Pri Due      Active Age    Description
   1             2/5/2011        3 secs Pay the mortgage
   2             2/5/2011        3 secs Pay the second mortgage
   3             2/5/2011             - Pay the second mortgage

  3 tasks
  $ task long

  ID Project Pri Added     Started Due      Recur  Countdown Age    Deps Tags Description
   1             1/29/2011         2/5/2011 1month   -6 days 6 secs           Pay the mortgage
   2             1/29/2011         2/5/2011 1month   -6 days 6 secs           Pay the second mortgage
   3             1/29/2011         2/5/2011 1month   -6 days 3 secs           Pay the second mortgage
   4             1/29/2011         2/5/2011 1month   -6 days      -           Pay the second mortgage

  4 tasks
  $ task long

  ID Project Pri Added     Started Due      Recur  Countdown Age    Deps Tags Description
   1             1/29/2011         2/5/2011 1month   -6 days 8 secs           Pay the mortgage
   2             1/29/2011         2/5/2011 1month   -6 days 8 secs           Pay the second mortgage
   3             1/29/2011         2/5/2011 1month   -6 days 5 secs           Pay the second mortgage
   4             1/29/2011         2/5/2011 1month   -6 days 2 secs           Pay the second mortgage
   5             1/29/2011         2/5/2011 1month   -6 days      -           Pay the second mortgage

  5 tasks

</pre>

This is specific to 1.9.4--or more exactly the source as of Jan 25 2011.
I cannot replicate this with 1.9.3.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2011-01-29T09:10:02Z
Modified: 2014-02-09T02:14:30Z
```

**Comment by taskwarrior** [comment]
_Cory Donnelly on 2011-01-30T17:49:16Z says:_ 

 Closed as duplicate of #656. I created unit tests for the two issues Steve mentions after seeing Bryce's bug report. Thanks guys.
